### PR TITLE
update docs url

### PR DESCRIPTION
**Pull request by jGaboardi**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

This PR updates the docs url.

### CODE REVIEW

**Comment by jGaboardi** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -47,7 +47,7 @@ about:
     The Spatial Interaction Modeling (SpInt) module provides a collection of
     tools to study spatial interaction processes and analyze spatial
     interaction data.
-  doc_url: http://readthedocs.org/projects/spint
+  doc_url: https://github.com/pysal/spint
```
Comment (martinfleis) [inline recipe/meta.yaml]:
```suggestion
  doc_url: https://spint.readthedocs.io/
```

no?

Comment (jGaboardi) [reply recipe/meta.yaml]: dang it. have not had my coffee yet.

Comment (jGaboardi) [reply recipe/meta.yaml]: And of course those docs [haven't been rebuilt since `v1.0.6`](https://spint.readthedocs.io/en/latest/?badge=latest)

**Review by martinfleis — APPROVED** [review]

**Pull request merged**